app/views/shared/check_answers/_merits.html.erb

```diff
@@ -97,11 +97,11 @@
       ) %>
 </dl>
 
-<div class='govuk-body'><%= simple_format statement_of_case.statement %></div>
+<div class='govuk-body' id="statement-of-case"><%= simple_format statement_of_case.statement %></div>
```

Comment:
See previous comment.

Reply:
I don't think this is needed at all anymore, so removed.